src/main/kotlin/org/rust/ide/annotator/fixes/AddMutableFix.kt

```diff
@@ -0,0 +1,46 @@
+package org.rust.ide.annotator.fixes
+
+import com.intellij.codeInspection.LocalQuickFixAndIntentionActionOnPsiElement
+import com.intellij.openapi.editor.Editor
+import com.intellij.openapi.project.Project
+import com.intellij.psi.PsiElement
+import com.intellij.psi.PsiFile
+import org.rust.lang.core.psi.*
+import org.rust.lang.core.psi.ext.RsNamedElement
+import org.rust.lang.core.psi.ext.parentOfType
+
+class AddMutableFix(val binding: RsNamedElement) : LocalQuickFixAndIntentionActionOnPsiElement(binding) {
+    override fun getFamilyName(): String = text
+    override fun getText(): String = "Add mutable"
```

Comment:
(Nitpicking) Personally I don't like naming this quickfix *Add mutable*. Maybe *Make variable mutable* or sth?